web/containers/tags-input-tag.js

```diff
@@ -11,16 +11,21 @@ export function TagsInputTag(props) {
     }
   });
 
+  const handleClick = useCallback(() => {
+    props.onRemoveTag(tag.data.tag);
+  }, [props, tag.data.tag]);
```

Comment:
I am not sure I follow, could it be a case when props or tag is changed. Is there any difference between using `useCallback` hook and using a closure, e.g:

```
<i className="ion-close-round" onClick={() => props.onRemoveTag(tag.data.tag)} />
```

Is using of `useCallback` hooks prevent re-rendering of `<i>` tag? Does it really matter? I mean I'd prefer better readability if it only affect extra rendering of single native node.